xml/art_slert_hw_testing.xml

```diff
@@ -2,26 +2,18 @@
 <?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook51-profile.xsl"
                  type="text/xml" 
                  title="Profiling step"?>
-<!DOCTYPE article
-[
-  <!ENTITY % entities SYSTEM "entity-decl.ent">
-  %entities;
-]>
-<!-- Converted by suse-upgrade version 1.1 -->
-
-<article version="5.0" xml:id="art-slert-hwtest" xml:lang="en"
-  xmlns="http://docbook.org/ns/docbook"
-  xmlns:xi="http://www.w3.org/2001/XInclude"
-  xmlns:xlink="http://www.w3.org/1999/xlink">
+<article xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:base="art_slert_hw_testing.xml" version="5.0" xml:id="art-slert-hwtest" xml:lang="en">
```

Comment:
Please don't change the header:
* Removing the DOCTYPE declaration with the entity include will make the file unbuildable.
* Adding xml:base to the root element is at least unnecessary.

Reply:
I updated and submitted as pull #18 request